pom.xml

```diff
@@ -3209,6 +3209,10 @@
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>2.22.2</version>
                     <configuration>
+                        <systemPropertyVariables>
+                            <!--https://issues.apache.org/jira/browse/JAMES-3724-->
+                            <james.ligecycle.leak.detection.mode>testing</james.ligecycle.leak.detection.mode>
+                        </systemPropertyVariables>
```

Comment:
Please carry this configuration change in projects where we overwrite surefire configuration.

EG mailbox/cassandra

Reply:
is it necessary? I  just checked, it inherits from parent pom, 
so these config properties are still available in child modules

Reply:
Ok then!